pvlib/test/test_modelchain.py

```diff
@@ -33,8 +33,8 @@ def cec_dc_snl_ac_system(sam_data):
     module_parameters['b'] = 0.05
     module_parameters['EgRef'] = 1.121
     module_parameters['dEgdT'] = -0.0002677
-    inverters = sam_data['cecinverter']
-    inverter = inverters['ABB__MICRO_0_25_I_OUTD_US_208_208V__CEC_2014_'].copy()
+    inverters = sam_data['adrinverter']
```

Comment:
Yea they are, I think the coverage is the same- this was something I was debating as well.